# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/nsIDOMNavigatorUserMedia.idl
# Commit: 98f0dd0b45cf
# Full Hash: 98f0dd0b45cf207837b59909bec2e253af15fecd
# Author: Paul Adenot <paul@paul.cx>
# Date: 2020-03-28 09:32:56
# Regressor Bug: 1624322
# File Overlap Count: 1
# Description:
#   Bug 1624322 - Add the raw group ID to MediaDevice. r=jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68082
# ==============================================================================

diff -r f3a28fba69d4 -r 98f0dd0b45cf dom/media/nsIDOMNavigatorUserMedia.idl
--- a/dom/media/nsIDOMNavigatorUserMedia.idl	Fri Mar 27 17:57:48 2020 +0000
+++ b/dom/media/nsIDOMNavigatorUserMedia.idl	Fri Mar 27 18:40:57 2020 +0000
@@ -14,6 +14,7 @@
   readonly attribute AString mediaSource;
   readonly attribute AString rawId;
   readonly attribute AString groupId;
+  readonly attribute AString rawGroupId;
   readonly attribute boolean scary;
   readonly attribute AString rawName; // unfiltered device name, from 1616661
 };
